test/integration/migrate/index.js

```diff
@@ -55,41 +54,16 @@ module.exports = function(knex) {
       });
 
       it('should return 0 if code matches DB', function() {
-        // Put in a couple dummy migrations. Needed
-        // since the migrations directory has a couple
-        // bad migrations that don't get loaded. This
-        // will simulate the DB and source being in sync.
-        return Promise.all([
-          knex('knex_migrations').returning('id').insert({
```

Comment:
This code was removed since it added mock data that is no longer required - this migration directory no longer has invalid migrations, they were moved to `test_with_invalid`.